app/controllers/two_factor_authentication/totp_verification_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
     end
 
     def create
-      result = TotpVerificationForm.new(current_user, params[:code].strip).submit
+      result = TotpVerificationForm.new(current_user, params.require(:code).strip).submit
```

Comment:
The return value of `require` is an `ActionController::Parameters`. Do we still need to access the property from it?

```suggestion
      result = TotpVerificationForm.new(current_user, params.require(:code)[:code].strip).submit
```

https://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-require